addons/mail/models/mail_presence.py

```diff
@@ -93,22 +89,23 @@ def _update_presence(self, inactivity_period, identity_field, identity_value):
     def _invalidate_im_status(self):
         self.user_id.invalidate_recordset(["im_status"])
         self.user_id.partner_id.invalidate_recordset(["im_status"])
+        self.guest_id.invalidate_recordset(["im_status"])
 
     def _send_presence(self, im_status=None, bus_target=None):
         """Send notification related to bus presence update.
 
         :param im_status: 'online', 'away' or 'offline'
         """
         for presence in self:
-            identity_data = presence._get_identity_data()
-            target = presence._get_bus_target()
-            target = bus_target or (target and (target, "presence"))
-            if identity_data and target:
-                self.env["bus.bus"]._sendone(
-                    target,
-                    "bus.bus/im_status_updated",
-                    {"im_status": im_status or presence.status, **identity_data},
-                )
+            self.env["bus.bus"]._sendone(
+                bus_target or (presence.guest_id or presence.user_id.partner_id, "presence"),
```

Comment:
Use bus listener mixin instead of _sendone

Reply:
I believe I can't since the bus target is the tuple `(id, "presence")`, or am I missing something?